OpenEdXMobile/res/values/profile_strings.xml

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!-- Profile Page -->
+    <!-- Label for wifi only switch -->
+    <string name="settings_wifi_top">Wifi only download</string>
+    <!-- Label explaining what wifi only means in detail -->
+    <string name="settings_wifi_bottom">Only download content when wi-fi is turned on</string>
+    <!-- Text to clarify that around what is visible -->
+    <string name="profile_visibility_off_message">Your profile information is only visible to you. Only your username is visible to others on {platform_name}.</string>
+
+    <!--  String for Profile Screen  -->
+    <!-- Button text to show user Video Settings screen -->
+    <string name="label_video_settings">Video Settings</string>
```

Comment:
We can move this to `video_strings.xml` too.

Reply:
moved.